bindings_ffi/examples/MainActivity.kt

```diff
@@ -1,29 +1,50 @@
 package com.example.xmtpv3_example
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
 import com.example.xmtpv3_example.R.id.selftest_output
 import kotlinx.coroutines.runBlocking
+import org.web3j.crypto.Credentials
+import org.web3j.crypto.ECKeyPair
+import org.web3j.crypto.Sign
+import uniffi.xmtpv3.FfiInboxOwner
+import java.nio.charset.StandardCharsets
+import java.security.SecureRandom
 
 const val EMULATOR_LOCALHOST_ADDRESS = "http://10.0.2.2:5556"
 
+class Web3jInboxOwner(private val credentials: Credentials) : FfiInboxOwner {
+    override fun getAddress(): String {
+        return credentials.address
+    }
+
+    override fun sign(text: String): ByteArray {
+        val messageBytes: ByteArray = text.toByteArray(StandardCharsets.UTF_8)
+        val signature = Sign.signPrefixedMessage(messageBytes, credentials.ecKeyPair)
+        return signature.r + signature.s + signature.v
```

Comment:
Nothing wrong here. But in the future we'll need to watch out which encoding the signing library is using for the V value. In this case bouncycastle is using eip155. but not always the case